modules/user/user.service.js

```diff
@@ -229,34 +224,51 @@ class UserService {
     return savedUser;
   }
 
+  async sendConfirmationLetter(email, language) {
+    const user = await this.getUserByFieldOrThrow('email', email);
+    const token = await generateToken(user._id, user.email);
+    const message = {
+      from: process.env.MAIL_USER,
+      to: user.email,
+      subject: 'Confirm Email',
```

Comment:
Please, rename subject to `[HORONDI] Email confirmation`, it is much more clear

Reply:
renamed